packs/st2/aliases/actions_list.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: "st2-actions-list"
```

Comment:
I don't think we should chatops this one. What is the point of showing stackstorm has these actions available they cannot be used.